definitions/npm/react-responsive_v9.x.x/flow_v0.83.x-/react-responsive_v9.x.x.js

```diff
@@ -0,0 +1,92 @@
+declare module 'react-responsive' {
+  declare export type MediaQueryTypes = {|
+    all?: boolean,
+    grid?: boolean,
+    aural?: boolean,
+    braille?: boolean,
+    handheld?: boolean,
+    print?: boolean,
+    projection?: boolean,
+    screen?: boolean,
+    tty?: boolean,
+    tv?: boolean,
+    embossed?: boolean,
+  |};
+
+  declare export type MediaQueryType = $Keys<MediaQueryTypes>;
+
+  declare export type MediaQueryMatchers = {|
+    aspectRatio?: string,
+    deviceAspectRatio?: string,
+    height?: number | string,
+    deviceHeight?: number | string,
+    width?: number | string,
+    deviceWidth?: number | string,
+    color?: boolean,
+    colorIndex?: boolean,
+    monochrome?: boolean,
+    resolution?: number | string,
+    orientation?: 'portrait' | 'landscape',
+    scan?: 'progressive' | 'interlace',
+    type?: MediaQueryType,
+  |};
+
+  declare export type MediaQueryFeatures = {|
+    ...MediaQueryMatchers,
+    minAspectRatio?: string,
+    maxAspectRatio?: string,
+    minDeviceAspectRatio?: string,
+    maxDeviceAspectRatio?: string,
+    minHeight?: number | string,
+    maxHeight?: number | string,
+    minDeviceHeight?: number | string,
+    maxDeviceHeight?: number | string,
+    minWidth?: number | string,
+    maxWidth?: number | string,
+    minDeviceWidth?: number | string,
+    maxDeviceWidth?: number | string,
+    minColor?: number,
+    maxColor?: number,
+    minColorIndex?: number,
+    maxColorIndex?: number,
+    minMonochrome?: number,
+    maxMonochrome?: number,
+    minResolution?: number | string,
+    maxResolution?: number | string,
+  |};
+
+  declare export type MediaQueryAllQueryable = {|
+    ...MediaQueryFeatures,
+    ...MediaQueryTypes,
+  |};
+
+  declare export type MediaQuerySettings = {|
+    ...MediaQueryAllQueryable,
+    query?: string,
+  |};
+
+  declare export var useMediaQuery: (
+    settings: MediaQuerySettings,
+    device?: MediaQueryMatchers,
+    onChange?: ((_: boolean) => void),
```

Comment:
```suggestion
    onChange?: ((matches: boolean) => void),
```
?

Reply:
Can fix, I was referring the TS definitions exactly and strangely that's what they had as their arg names

Reply:
Okay thanks, as you wish ;)